src/gridfs/index.ts

```diff
@@ -109,7 +111,10 @@ export class GridFSBucket extends TypedEventEmitter<GridFSBucketEvents> {
     filename: string,
     options?: GridFSBucketWriteStreamOptions
   ): GridFSBucketWriteStream {
-    return new GridFSBucketWriteStream(this, filename, options);
+    return new GridFSBucketWriteStream(this, filename, {
+      timeoutMS: this.s.db.timeoutMS,
```

Comment:
We stored timeoutMS to this bucket but then reach to the db for the value, is that intentional?

Reply:
This was an oversight. Good catch.